File: JavaScript-Fundamentals/09.Strings/04.Parse tags/Parse tags.js
/** Parse tags
 *You are given a text. Write a function that changes the text in all regions:

 <upcase>text</upcase> to uppercase.
 <lowcase>text</lowcase> to lowercase
 <orgcase>text</orgcase> does not change casing
 */
function solve(args) {
    var text = args[0];

    text = text.replace();
    console.log(text);
}
solve(['We are <orgcase>liViNg</orgcase> in a <upcase>yellow submarine</upcase>. ' +
'We <orgcase>doN\'t</orgcase> have <lowcase>anything</lowcase> else.']);

File: Homeworks/1. Programming/2. C#-Part-2/06.Strings-and-Text-Processing/05.Parse tags/ParseTags.cs
//You are given a text.Write a program that changes the text in all regions surrounded by the tags<upcase> and</upcase> to upper-case.

namespace _05.Parse_tags
{
    using System;
    using System.Text.RegularExpressions;

    class ParseTags
    {
        static void Main()
        {
            var text = Console.ReadLine();

            Console.WriteLine(Regex.Replace(text, "<upcase>(.*?)</upcase>", m => m.Groups[1].Value.ToUpper()));
        }
    }
}
